index.php

```diff
@@ -595,11 +601,11 @@
 	$checks['version_check_status'] = '<span class="pass">'.$lang->requirementspage_uptodate.'</span>';
 
 	// Check for a new version of the Merge System!
-	require_once MYBB_ROOT."inc/class_xml.php";
+	require_once MYBB_ROOT."inc/class_xmlparser.php";
 	$contents = merge_fetch_remote_file("http://www.mybb.com/merge_version_check.php");
 	if($contents)
 	{
-		$parser = new XMLParser($contents);
+		$parser = new MyBBXMLParser($contents);
```

Comment:
We may call [`create_xml_parser($contents)`](https://github.com/mybb/mybb/blob/mybb_1829/inc/functions.php#L9417) to get a XML parser for `$contents`.